tests/commands/test_commands.py

```diff
@@ -78,26 +78,6 @@ def test_bigchain_show_config(capsys):
     assert output_config == config
 
 
-@pytest.mark.tendermint
-def test_bigchain_run_init_when_db_exists(mocker, capsys):
```

Comment:
I guess this test could re-purposed to asset a success message?

Reply:
This particular test does not use a real database but asserts that one function catches an exception raised by the other.

Reply:
There is a pattern of not using the real database in this file so I would leave it to be handled by  future test (re)organisation efforts.